### DISCUSSION TITLE
Issues integrating into a virtual environment

### DISCUSSION THREAD

**Discussion by lonely-neutrino** [posit-dev/great-tables#524] [Q&A]
I've been having issues using this in virtual environments in anaconda. Even creating a new conda environment around it with `conda create -n greatTables_env great_tables` (I also tryed pip installing but that was even more troublesome). I am then unable to install pandas with conda and there is a conflict. Additionally, I can't even install spyder. For me it's pretty hard to use a package that conflicts with basic things like pandas and spyder, which is a shame because great_tables looks like such a cool package. Perhaps, I am doing something wrong so any suggestions would be appreciated. Thanks!

**Comment by kamilm** [comment]
Hi, I was puzzled by your comment and tried it myself, I was able to install the packages you mentioned in 2 or three steps:

`conda create --name tmp-test python=3.12 pip spyder pandas`

`conda activate tmp-test`

`conda install -c conda-forge great_tables`

this worked, took a few moments as conda is slow, but it worked none the less. I needed to used `conda-forge` as I wasn't able to find `great_tables` on the default channel. 

I hope this helps.
